src/Modules/UngradedModules/ModuleGens.jl

```diff
@@ -325,10 +346,8 @@ function lift(a::FreeModElem{T}, generators::ModuleGens{T}) where {T <: MPolyRin
   if iszero(a)
     return sparse_row(base_ring(parent(a)))
   end
-  singular_assure(generators)
   S = singular_generators(generators)
   b = ModuleGens([a], generators.SF)
```

Comment:
```suggestion
  b = ModuleGens([a], singular_freemodule(generators))
```
Doesn't this need an explicit call since the assure two lines above was removed? (I assume that singular_generators in the line above will populate the SF field, but that is just a side-effect of that function and not documented)

Reply:
Thx for catching, missed that one, we can of course use the new getter.